src/client/sandbox/event/simulator.ts

```diff
@@ -605,7 +605,9 @@ export default class EventSimulator {
     }
 
     _dispatchMouseEvent (el, args, { dataTransfer, timeStamp }: any) {
-        const disabledParent = domUtils.findParent(el, true, node => node.disabled);
+        const disabledParent = domUtils.findParent(el, true, node => {
+            return domUtils.isElementNode(node) && nativeMethods.hasAttribute.call(node, 'disabled');
```

Comment:
`domUtils.isElementNode(node)` condition is unnecessary

`const disabledParent = domUtils.findParent(el, true, node => nativeMethods.hasAttribute.call(node, 'disabled'));`

Reply:
In the current `findParent` implementation the `document` element can be parent. Document does not have the `hasAttribute` method